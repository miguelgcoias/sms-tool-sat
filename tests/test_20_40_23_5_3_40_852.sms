20
17 14 31 5 3 4 2 2 3
35 19 54 4 5 4 5 5
23 13 36 2 6 7
0 9 9 1 9
11 19 30 1 19
0 12 24 1 12
12 9 30 1 9
39 20 99 4 7 3 6 4
22 4 26 2 2 2
36 5 46 5 1 1 1 1 1
10 9 19 4 3 2 3 1
39 2 43 1 2
15 9 42 4 1 4 2 2
6 16 22 2 7 9
8 16 40 3 6 6 4
24 13 37 4 3 2 3 5
14 10 44 5 1 2 2 2 3
34 6 40 3 1 2 3
11 16 43 3 5 4 7
2 21 44 1 21
0
3 1 20 4
0
0
0
0
0
2 14 16
1 7
2 4 11
0
3 17 20 18
2 20 5
0
0
0
2 11 1
2 11 17
3 13 4 15
0
