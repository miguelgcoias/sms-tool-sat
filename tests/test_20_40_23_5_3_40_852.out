4
4 0
7 19
10 36 37 38 39 40
11 10 13 15 18
